src/olympia/editors/templates/editors/includes/files_view.html

```diff
@@ -1,5 +1,6 @@
 <div><strong>{{ _('Files in this version:') }}</strong></div>
 <ul>
+{% set latest_version=addon.find_latest_version(channel=amo.RELEASE_CHANNEL_LISTED) %}
```

Comment:
`version` is passed to this template so I can use what I wrote in the comment above.